src/unix/notbsd/android/mod.rs

```diff
@@ -87,6 +88,13 @@ s! {
         pub l_len: ::off_t,
         pub l_pid: ::pid_t,
     }
+
+    pub struct cpu_set_t {
+        #[cfg(target_pointer_width = "64")]
+        __bits: [__CPU_BITTYPE; 16],
+        #[cfg(target_pointer_width = "32")]
+        __bits: [__CPU_BITTYPE; 1],
+    }
```

Comment:
Possibly... I had a short span of time over the weekend and strongly suspect I merged a change into the wrong PR...